src/LMS1xx.cpp

```diff
@@ -187,6 +187,18 @@ void LMS1xx::setScanDataCfg(const scanDataCfg &cfg) {
 	buf[len - 1] = 0;
 }
 
+void LMS1xx::setOutputRange(const scanOutputRange &cfg) {
+	char buf[100];
+	sprintf(buf, "%c%s 1 %X %X %X%c", 0x02, "sWN LMPoutputRange",
+			cfg.angleResolution, cfg.startAngle, cfg.stopAngle, 0x03);
+
+	write(sockDesc, buf, strlen(buf));
+
+	int len = read(sockDesc, buf, 100);
```

Comment:
What's the meaning of the third param of the `read` function? I'm asking because depending on that you should have `buf[101]` instead of `buf[100]`, because the `buf[len - 1] = 0` might overwrite the latest value read if `read` reads `100` characters.